stable/insights-agent/templates/right-sizer/controller-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.rightsizer.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-right-sizer-controller
+  annotations:
+  labels:
+    component: right-sizer
+    app: insights-agent
+spec:
+  selector:
+    matchLabels:
+      component: right-sizer
+      app: insights-agent
+  template:
+    metadata:
+      labels:
+        component: right-sizer
+        app: insights-agent
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}-right-sizer-controller
+          ports:
+          - containerPort: 8080
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.rightsizer.image.repository }}:{{ .Values.rightsizer.image.tag }}"
+          imagePullPolicy: {{ .Values.rightsizer.image.pullPolicy }}
+          command:
+          - "right-sizer"
+          # NOt yet supported by the controller:
+          #- "-stateconfigmap"
+          #- "{{ .Release.Namespace }}/{{ .Values.rightsizer.stateconfigmapname }}"
```

Comment:
You will also need to add health check port ... otherwise `kubehunter` scan will be complaining

Reply:
> You will also need to add health check port ... otherwise `kubehunter` scan will be complaining

Making sure I understand - do you mean that I need to supply a readiness, or liveness, or both probes? The current release of the controller does not have those, but I can include one or both of them in the next release.

Reply:
@ivanfetch-fw there is a check to make sure both  `readiness` and `liveness` are available. including them in the next release should be fine. thanks.